Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6883,6 +6884,17 @@ WebShareFileAPIEnabled:
     WebCore:
       default: false
 
+WebSocketsEnabled:
```

Comment:
I'm not sure we really need this setting flag. Can modern websites really work without WebSockets?